rwdavies/motif_death
# Update config to be set at SPECIES_ORDER, RUN_ID and HATBAG_OUTPUT_DIR level

Goal: Change config so each run is set at SPECIES_ORDER, RUN_ID (i.e. which species you are running) and HATBAG_OUTPUT_DIR level.  See README for writeup.

Test passed
Artiodactyla rerun with cow data removed 

Additional changes:
* Can specify number of cores to use when running `./scripts/test.sh`, see README
* Hatbag able to run on Artiodactyla
* Change some treemix, mapping jobs to e nodes to prevent bash illegal instruction error when running on D nodes
* Remove duplicated / unnecessary code in `R/get_genome_stats.R` and `R/get_per_sample_average_cov.R`
